### ISSUE TITLE
Breaks SCSS with if/else

### ISSUE THREAD

**Issue by altschuler** [prettier/prettier#2591]
This
```css
@function _normalize-include($section) {
  // Check if $section is in the $include list.
  @if index($_normalize-include, $section) {
    @return true;
  }
  // If $include is set to (all), make sure $section is not in $exclude.
  @else if not index($_normalize-exclude, $section) and index($_normalize-include, all) {
    @return true;
  }
  @return false;
}
```

Gets emitted as 

```css
@function _normalize-include($section) {
  // Check if $section is in the $include list.
  @if index($_normalize-include, $section) {
    @return true;
  }
  // If $include is set to (all), make sure $section is not in $exclude. @else if not index($_normalize-exclude, $section) and index($_normalize-include, all) {
    @return true;
  }
  @return false;
}
```

The `@else` becomes part of the comment

**Comment by alexilyaev** [comment]
Got the same exact issue :-)

**Comment by azz** [comment]
Fixed in #2600
